src/app/components/analytics/host-analytics/host-analytics.component.scss

```diff
@@ -0,0 +1,60 @@
+li.analytics-list {
+  margin-top: 25px;
+}
+
+.analytics-challenge-title {
+  margin-bottom: 20px;
+}
+
+.analytics-team-input {
+  input[type="radio"].selectTeam + label {
+    padding-left: 0px;
+    width: 100%;
+  }
+}
+
+div.show-member-title {
+  width: calc(100% - 35px);
+  float: right;
+}
+
+.show-analytics-btn {
+  margin-top: 30px;
+}
+
+h4.analytics-details-head {
+  margin-top: 2px;
+}
+
+.analytics-image {
+  img {
+    width: 100%;
+  }
+}
+
+.analytics-card-content {
+  h3 {
+    margin-bottom: 2px;
+  }
+}
+
+.analytics-phases-card {
+  margin-top: 10px;
+}
+
+
+
+
+.participant-download-btn {
+ margin-top: 0px !important;
+}
+
+.analytics-challenge-single-line{
+  display:flex;
+  flex-direction: row;
+  .show-count {
+    margin-left: 5px !important;
```

Comment:
Do we really need **!important** here? (Also, at other places.)

Reply:
Yeah, Also using **!important** inside component specific css is toatally safe. It do not hinder with other css

Reply:
It does not hinder but overrides, that might cause sometimes unexpected behavior. I meant that we should only use this when we really need it.

Reply:
It overrides for component specific html only that's why it's safe